sos/report/plugins/pulp.py

```diff
@@ -18,7 +18,7 @@ class Pulp(Plugin, RedHatPlugin):
     short_desc = 'Pulp platform'
 
     plugin_name = "pulp"
-    packages = ("pulp-server", "pulp-katello")
+    packages = ("pulp-server", "pulp-katello", "python3-pulpcore")
```

Comment:
I don't know sos that well. Does this mean that if any of these packages is installed, it runs?

Reply:
Yes. Presence of either package will trigger this plugin automatically / by default.